Veradux/wwa
# Update the fork with the changes from the main repo (#15)

* Create .travis.yml

* updated pom.xml with codecov plugin

* updated .travis.yml with script for codecov

* adding codecov file

* added com.sum to pom.xml

https://stackoverflow.com/questions/51503460/could-not-find-artifact-com-suntoolsjar1-7/51616063#51616063

* Fixed pom.xml

Had to remove the entire <dependency> block, but now it builds.

* fix: updated pom.xml and redid package structure

* testing codecov

* fix: this commit fixes the mainclass pom tag and fixes npes in main

* stop travis from checking in JDK11

* Remove CONTIBUTING template and describe actual PR flow

* add shield to readme

* Code changes

* Link name fix

* Added Entity Class with all fields we defined in UML , created Ant,Wasp and Spider classes which inherited from entity class, created QueenAnt and WorkerAnt class which inherited from Ant class. Remove game and player.java as those didnt had need. Removed getters  and setters as these were not needed in inheritance environment.

* Added Entity Class with all fields we defined in UML , created Ant,Wasp and Spider classes which inherited from entity class, created QueenAnt and WorkerAnt class which inherited from Ant class. Remove game and player.java as those didnt had need. Removed getters  and setters as these were not needed in inheritance environment.

* Updating README

I want to update this on the master repo so people visiting for the first time see it. Added logo, re-write first section, removed Motivation (redundant after changes) and How to Play sections (will add How to Play back when there's something to play), added new FAQ section and new collapsible list formatting throughout.

* Removed multiplayer reference, removed JDK11 references, updated IDE recommendations

* Fixed Readme Badges

Centered Badges, added link to Contributors badge, added discord channel link to discord badge, added discord logo to discord badge.
